prql-compiler/prqlc/src/cli.rs

```diff
@@ -149,6 +149,10 @@ pub enum DebugCommand {
 
     /// Parse, resolve & combine source with comments annotating relation type
     Annotate(IoArgs),
+
+    /// Print info about the AST data structure
+    #[cfg(debug_assertions)]
+    Ast,
```

Comment:
I really like this idea — it's easy to debug this time, and _really_ easy to see diagnostics in the future — adding Structs here is v simple...

Reply:
:+1:

... and I forgot `debug_assertions` in. This will not compile with `--release` flag.

Reply:
I thought that was intentional — I thought it was a good idea too!! 

...given they're used when developing rather than "using" (though also we should avoid having lots of "Modes")

Reply:
Yea, the `#[cfg(...)]` should be removed, as I said.

Reply:
Ah OK, so not having lots of "Modes' wins out here...